docs/supported_apis/series_supported.rst

```diff
@@ -118,25 +122,27 @@ the related section on :doc:`Defaulting to pandas </supported_apis/index>`.
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``dot``                     | Y                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``drop``                    | Y                               |                                                    |
+| ``drop``                    | Y                               | **Omnisci**:``P``, ``D`` for row drop?             |

```

Comment:
there is nothing we can drop in `Series` except for rows, so the function should be `D`

Reply:
Right